eWonConnector-gateway/src/main/java/com/hms_networks/EwonUtil.java

```diff
@@ -185,9 +185,9 @@ public static String httpGet(String url) throws Exception {
 
          // Generate a "cleaned" exception based on the response code.
          if (httpResponse == HTTP_UNAUTHORIZED) {
-            logger.error("Verify your account name, password, and developer ID are correctly"
-                  + " entered on the Ewon Connector configuration page");
-            logger.error("Talk2M account credentials are invalid!");
+            logger.error("Your Talk2M account credentials are invalid! " +
```

Comment:
Maybe state that the configured / entered credentials are not valid.

This error makes it sound like their actual credentials are incorrect.

Reply:
changed